### PR TITLE
Add question consensus reducer

### PR DESCRIPTION
**Pull request by alnah005**
Background:
The often-used consensus reducer for Caesar would be a nice addition for folks to be able to use for offline analysis. See issue #277 

Purpose:
Return the question option with the most votes.

Outputs:
most_likely = option with greatest number of classifications/votes
num_votes = vote count for mostly likely option
agreement = fraction of total votes held by most likely option.

### CODE REVIEW

File: panoptes_aggregation/reducers/question_consensus_reducer.py
Comment (CKrawczyk) [inline panoptes_aggregation/reducers/question_consensus_reducer.py:50]: You can do this with `counter_total.most_common(1)` to get the max in easier to read code. (`Counter` have many nice methods for this kind of thing).

File: panoptes_aggregation/reducers/question_consensus_reducer.py
Comment (CKrawczyk) [inline panoptes_aggregation/reducers/question_consensus_reducer.py:58]: What part of the above code is likely to give a `ValueError`?  Can an `if` block be used instead? `try` blocks tend to be slower than `if` and the speed might become an issue when trying to reduce large numbers of classifications.

**Review by CKrawczyk — COMMENTED** [review]
Looks good, just some comments to improve the readability of the code.

File: panoptes_aggregation/reducers/question_consensus_reducer.py
Comment (alnah005) [reply panoptes_aggregation/reducers/question_consensus_reducer.py:50]: I like your suggestion. I'll add that!

File: panoptes_aggregation/reducers/question_consensus_reducer.py
Comment (alnah005) [reply panoptes_aggregation/reducers/question_consensus_reducer.py:58]: It's actually the max. Now that I'm switching it to `counter_total.most_common(1)`, having an if statement will suffice!

**Review by CKrawczyk — APPROVED** [review]
The code looks good to me.  @alnah005 do you have permissions to merge to master, or should I do that?  

Once merged to master the code will auto-deploy and be ready to use with Caesar, because of this we should hold off merging until Monday.

**Comment by alnah005** [comment]
> The code looks good to me. @alnah005 do you have permissions to merge to master, or should I do that?
> 
> Once merged to master the code will auto-deploy and be ready to use with Caesar, because of this we should hold off merging until Monday.

I don't have permissions to merge to master, probably need your help for that.

Monday works for me!

**Pull request merged**
